.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "dependencies"
+      - "bot"
```

Comment:
@thaJeztah I use arbitrary labels. Let me know what we should use.

Reply:
Perhaps the `bot` label is not needed.

If this is only updating the CI actions, perhaps;

- `area/testing` would be ok (I tend to use that for fixes in tests and CI)
- `status/2-code-review`


Do you know if there's now a way to make dependant only run on this repository, and not on forks? (e.g. can we restrict this on repository name?)

Reply:
> Do you know if there's now a way to make dependant only run on this repository, and not on forks? (e.g. can we restrict this on repository name?)

Doesn't look like it :disappointed:
* https://github.com/dependabot/dependabot-core/issues/2198
* https://github.com/dependabot/dependabot-core/issues/2804

Reply:
I was hoping because this one is a schedule; perhaps we could disable the schedule if repo != docker/cli (which would be slightly different than a default dependabot check?). But yeah, if it's not possible, then, well, perhaps it is what it is (and it'll just open 1500 useless pull requests in all forks 😂)